### PR TITLE
Codeformatting

### PR DESCRIPTION
**Pull request by iranzo**

### CODE REVIEW

**Comment by karmab** [comment]
i like the python code formatting stuff, but let's deal with the shebang stuff in another PR

File: kvirt/baseconfig.py [hunk 1]
```diff
@@ -326,10 +326,9 @@ def bootstrap(self, name, host, port, user, protocol, url, pool, poolpath):
             poolpath = '/var/lib/libvirt/images'
         if host == '127.0.0.1':
             ini = {'default': {'client': 'local', 'cloudinit': True,
-                               'tunnel': False, 'reservehost': False,
-                               'insecure': True, 'enableroot': True,
+                               'tunnel': False, 'insecure': True, 'enableroot': True,
                                'reserveip': False, 'reservedns': False,
-                               'reservehost': False, 'nested': True,
+                               'nested': True,
```
Comment (karmab) [inline kvirt/baseconfig.py]: why did you remove this parameters?

File: kvirt/baseconfig.py [hunk 2]
```diff
@@ -338,12 +337,10 @@ def bootstrap(self, name, host, port, user, protocol, url, pool, poolpath):
...
                 name = host
             ini = {'default': {'client': name, 'cloudinit': True,
-                               'tunnel': True, 'reservehost': False,
-                               'insecure': True, 'enableroot': True,
+                               'tunnel': True, 'insecure': True, 'enableroot': True,
                                'reserveip': False, 'reservedns': False,
-                               'reservehost': False, 'nested': True,
-                               'start': True}}
-            ini[name] = {'host': host, 'pool': pool, 'nets': ['default']}
+                               'nested': True,
```
Comment (karmab) [inline kvirt/baseconfig.py]: really dont like reservehost keyword ?

Comment (iranzo) [reply kvirt/baseconfig.py]: done

File: kvirt/openstack/__init__.py [hunk 3]
```diff
@@ -142,7 +142,7 @@ def create(self, name, virttype='kvm', profile='', plan='kvirt', flavor=None,
             flavor = flavors[0] if flavors else nova.flavors.find(name="m1.tiny")
             common.pprint("Using flavor %s" % flavor.name, color='green')
         elif flavor not in allflavornames:
-            return {'result': 'failure', 'reason': "Flavor % not found" % flavor}
+            return {'result': 'failure', 'reason': "Flavor %ds not found" % flavor}
```
Comment (karmab) [inline kvirt/openstack/__init__.py]: you meant %s

Comment (iranzo) [reply kvirt/openstack/__init__.py]: yup, fixed
